documentation/asciidoc/topics/ref_encryption_service_ca.adoc

```diff
@@ -0,0 +1,36 @@
+[id='encryption_with_service_ca-{context}']
+= Encryption with {openshift} Service Certificates
+{ispn_operator} automatically generates TLS certificates that are signed by the
+{openshift} service CA. {ispn_operator} then stores the certificates and keys
+in a secret so you can retrieve them and use with remote clients.
+
+If the {openshift} service CA is available, {ispn_operator} adds the following
+`spec.security.endpointEncryption` configuration to the **Infinispan** CR:
+
+[source,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  security:
+    endpointEncryption:
+      type: service
+      certServiceName: service.beta.openshift.io <1>
+      certSecretName: {example_crd_name}-cert-secret <2>
+----
+
+<1> Specifies the {openshift} Service.
+<2> Names the secret that contains a service certificate, `tls.crt`, and key, `tls.key`, in PEM format. By default, {ispn_operator} appends `-cert-secret` to the {brandname} cluster name.
```

Comment:
@oraNod maybe a little bit clear related default: If this name is not defined, {ispn_operator} expects following secret name _{brandname} cluster name_ `-cert-secret`.